roles/libvirt_manager/tasks/create_vms.yml

```diff
@@ -21,17 +21,20 @@
     _workload: "{{ cifmw_libvirt_manager_basedir }}/workload"
     _img_dir: "{{ cifmw_libvirt_manager_basedir }}/images"
     _chdir: >-
-      {{ (is_base_img | default(false) | bool) | ternary(_img_dir, _workload) }}
+      {{ (_is_base_img | default(false) | bool) | ansible.builtin.ternary(_img_dir, _workload) }}
+    _is_ocp_vm: "{{ vm is match('.*ocp.*')}}"
   block:
     - name: "Create VM image for {{ vm }}"
-      vars:
-        _vm_img: >-
-          {{ vm }}.qcow2
+      # when:
+      #   - (_is_ocp_vm and cifmw_use_ocp_overlay | default(false) | bool) or
+      #     not _is_ocp_vm
       ansible.builtin.command:
         cmd: >-
           qemu-img create
           {% if vm_data.disk_file_name != 'blank' %}
-          -o backing_file={{ _img }},backing_fmt=qcow2
+            {% if (_is_ocp_vm and cifmw_use_ocp_overlay | default(false) | bool) or not _is_ocp_vm -%}
```

Comment:
I don't think this is needed. the `creates: {{ _vm_img }}` should do it properly already. What you want is to ensure the "overlay-less" OCP cluster is deployed so that the `_vm_img` matches. To do that, I think you should toggle the value of `_is_base_img` set in the `ocp_layout.yml` (in libvirt_manager role iirc).

Reply:
Done